logic/peerSweeper.js

```diff
@@ -5,18 +5,18 @@ var pgp = require('pg-promise');
 // Constructor
 function PeerSweeper (scope) {
 	this.peers = [];
-	this.limit = 100;
+	this.limit = 500;
 	this.scope = scope;
 
 	var self = this;
 
 	setImmediate(function nextSweep () {
 		if (self.peers.length) {
 			self.sweep(self.peers.splice(0, self.limit), function () {
-				return setTimeout(nextSweep, 1000);
+				return setTimeout(nextSweep, 500);
```

Comment:
They are now same value, but mean different things. `this.limit` is number of peers. However this is a time interval for next sweep.